web/src/pages/documents/documents-page.tsx

```diff
@@ -95,14 +95,15 @@ const DocumentsPage = () => {
                                 onDragOver={() => setFileOver(true)}
                                 onDragLeave={() => setFileOver(false)}
                                 onDrop={() => setFileOver(false)}
-                                style={{ height: '100%', overflow: 'hidden' }}
+                                style={{
+                                    height: '100%',
+                                    overflow: 'hidden',
+                                    display: 'flex',
+                                    flexDirection: 'column'
+                                }}
                             >
-                                {!isFileOver && (
-                                    <div
-                                        style={{
-                                            height: '100%'
-                                        }}
-                                    >
+                                <DocumentsDropZone dataset={activeDataset}>
```

Comment:
Do you know how should drag & drop work when we move file via "in documents" tab? I've tried to move usual jpg there, it was successfully loaded but shown in "in file names" tab. Should we somehow automatically redirect the user to the place where it was placed? Or it should appear in "in documents" tab, do you know? Is drag & drop actually needed for "in documents" tab or not?

Reply:
Actually, I don't know how it should work. Anyway, it's out of scope of this PR

Reply:
Okay, moved this to "Open questions"